vscode/src/chat/protocol.ts

```diff
@@ -90,6 +90,16 @@ export type WebviewMessage =
           command: 'attribution-search'
           snippet: string
       }
+    | {
+          command: 'troubleshoot/reloadAuth'
+      }
+    | {
+          command: 'troubleshoot/showOutputLogs'
+      }
+    | {
+          command: 'troubleshoot/fullReset'
+          askConfirmation?: boolean
+      }
```

Comment:
```suggestion
    | {
          command: 'troubleshoot'; action: 'output' | 'reset'
      }
```

Can this be simplified? We already have a [reload](https://sourcegraph.com/github.com/sourcegraph/cody/-/blob/vscode/src/chat/protocol.ts?L74) command so i think we just need to add 'reset' and `output`?
We should always ask for confirmation before resetting, imo.